src/components/20-molecules/datepicker/index.js

```diff
@@ -355,21 +369,21 @@ class AXADatepicker extends NoShadowDOM {
 
                   <div class="m-datepicker__weekdays">
                     ${this.weekdays &&
-                      this.weekdays.map(
-                        day =>
-                          html`
+      this.weekdays.map(
```

Comment:
here looks broken with tabs / new lines

Reply:
Oh yes thanks... i have used intelliJ formatter, not prettier